[ID] Add Beatmap Nominators and Modding group articles

hikikuman opened the pull request:
Added Indonesian translation for Beatmap Veto, Becoming a Beatmap Nominator, Expectations, General Information (also img files inside), and Rules.

hikikuman commented:
Updated Indonesian translation for Modding. Added Indonesian translation for BAT, BATm, MAT, MAT Leaders, Kudosu, M4M, NM, Mod request, Modder, NAT Meetings, Nuke, Proto-bubble, Star priority, and Timestamp.

Nivalyx commented on wiki/Modding/Mod_for_mod/id.md:
```diff
@@ -0,0 +1,15 @@
+
+# Mod for mod
+
+**Mod for mod** (juga dikenal sebagai *M4M*) adalah sebuah istilah yang menggambarkan kesepakatan antara dua kreator atau lebih. Setiap kreator harus melakukan [mod](/wiki/Modding) pada [beatmap](/wiki/Beatmaps) pengguna lain.
```
`kreator` feels really weird here to be honest, given the context here using `mapper` would be a lot better

``**Mod for mod** (juga dikenal sebagai *M4M*) merupakan istilah yang menggambarkan kesepakatan antara dua orang mapper atau lebih, di mana masing-masing mapper sepakat untuk melakukan [modding](/wiki/Modding) terhadap [beatmap](/wiki/Beatmaps) milik mapper lain yang terlibat.``

hikikuman replied:
i agree. ok, changed

Nivalyx commented on wiki/Modding/Mod_request/id.md:
```diff
@@ -0,0 +1,9 @@
+
+# Mod request
+
+**Mod request** adalah istilah yang digunakan ketika seorang kreator membutuhkan bantuan dan ingin memperbaiki [beatmap](/wiki/Beatmaps) mereka. Hal ini dapat dilakukan dengan banyak cara, tetapi paling sering dilakukan dengan meminta normal mod atau [mod for mod](/wiki/Modding/Mod_for_mod) pada salah satu [queue di forum modding](https://osu.ppy.sh/community/forums/60). Setelah permintaan mod di ...
```
same here, `kreator` --> `mapper`

Nivalyx commented on wiki/Modding/NAT_meetings/id.md:
```diff
@@ -0,0 +1,40 @@
+
+# Pertemuan NAT
+
+**Pertemuan NAT** adalah diskusi terarah yang diselenggarakan oleh [Nomination Assessment Team](/wiki/People/The_Team/Nomination_Assessment_Team) (*NAT*) di channel `#modding` pada [server Discord osu!dev](/wiki/osu!dev_Discord_server). Topik pertemuan berkaitan dengan ekosistem [mapping](/wiki/Beatmapping) dan [modding](/wiki/Modding), termasuk masalah, solusi yang berpotensi, ide baru, dan lapo ...
```
uhhhh i don't think `Meeting` here translates into `Pertemuan` tho, something like `Rapat Terbuka` or even `Musyawarah` is much more appropriate :(

``**Rapat Terbuka NAT** merupakan suatu diskusi terarah yang diselenggarakan oleh [Nomination Assessment Team](/wiki/People/The_Team/Nomination_Assessment_Team) (*NAT*) di channel `#modding` pada [server Discord osu!dev](/wiki/osu!dev_Discord_server). Setiap rapat yang diselenggarakan pada umumnya mengangkat topik bahasan yang berkaitan dengan ekosistem [mapping](/wiki/Beatmapping) dan [modding](/wiki/Modding) yang di dalamnya mencakup antara lain berbagai permasalahan terkini, solusi, ide-ide baru, dan laporan perkembangan. Meskipun rapat ini diutamakan bagi anggota NAT atau [Beatmap Nominator](/wiki/People/The_Team/Beatmap_Nominators), siapapun diperkenankan untuk ikut serta di dalamnya.``

Nivalyx commented on wiki/Modding/NAT_meetings/id.md:
```diff
@@ -0,0 +1,40 @@
+- BN activity/reset quantification for efficient evaluation (proposal tertunda)
+- If/how/when reset severity applies to general map quality
+
+### Topik yang dibatalakan
```
typo, ``dibatalakan`` --> ``dibatalkan``

hikikuman replied:
uhh my bad

Nivalyx commented on wiki/Modding/Normal_mod/id.md:
```diff
@@ -0,0 +1,13 @@
+
+# Normal mod
+
+**normal mod** (*NM*) adalah sebuah [mod](/wiki/Modding) yang diberikan tanpa perlu melakukan pekerjaan apa pun kepada kreator yang memberikan mod tersebut. Normal mod biasanya dilakukan untuk mendapatkan pendapat kedua pada [beatmap](/wiki/Beatmaps) untuk membantu memperbaikinya. Banyak modder yang menggunakan istilah "NM" untuk membedakannya dari [mod for mod](/wiki/Modding/Mod_for_mod).
```
``Normal`` should be capitalized

also `kreator` -> `mapper` as before

hikikuman replied:
whaa ok2

Nivalyx reviewed (commented):
segini dulu deh, ini banyak banget yang mau di-push di satu pull request aaaaaaaaa :(

hikikuman commented on wiki/Modding/Kudosu/id.md:
```diff
@@ -0,0 +1,23 @@
+
+Kata *kudosu*, terkadang ditulis dengan *kudosu!*, 
+
+The word *kudosu*, sometimes stylised as *kudosu!*, adalah [penggabungan dua kata baru](https://en.wikipedia.org/wiki/Portmanteau) dari *kudos* dan *osu!*.
```
whaa my bad

rorre commented on wiki/People/The_Team/Beatmap_Nominators/Rules/id.md:
```diff
@@ -0,0 +1,76 @@
+
+Aktivitas diukur dengan nominasi, yang berarti setiap nominasi bernilai satu poin aktivitas. Setiap pemeriksaan QA sama dengan 1/4 aktivitas dari sebuah nominasi. Mereset beatmap tetap diperhitungkan tetapi pada tingkat yang lebih rendah, dan tidak ada rasio yang ditetapkan untuk dibandingkan dengan nominasi.
+
+Setiap mode memiliki kebutuhan minimumnya sendiri untuk aktivitas bulanan.
```
```suggestion
Setiap mode memiliki kebutuhan minimum tersendiri untuk aktivitas bulanan.
```

rorre commented on wiki/People/The_Team/Beatmap_Nominators/Rules/id.md:
```diff
@@ -0,0 +1,76 @@
+- **Jangan menominasi beatmapset yang terdapat kontribusi dari kamu.** Hal ini berlaku untuk beatmap kamu sendiri, keikutsertaan dalam kolaborasi pada tingkat kesulitan, serta beatmap yang kamu buat hitsound atau storyboardnya.
+- **Jangan menominasi sebuah beatmapset tanpa memeriksa semua tingkat kesulitan dari suatu mode dengan benar.** Pemeriksaan yang ceroboh seperti hanya menjalankan alat modding saja sebelum menominasikan atau bahkan tidak mengunduh beatmap.
+- **Jangan menominasi sebuah beatmap jika kamu tidak yakin terhadap sesuatu.** Dalam kasus seperti itu, mintalah bantuan dari sesama nominator sebelum membuat keputusan yang terburu-buru. Ini termasuk mode permainan yang tidak dikenal, sesuatu yang mencurigakan/masalah unrankable, masalah kualitas atau timing yang rumit.
+- **Jangan menominasi sebuah beatmap jika kamu tidak dapat meninainya dengan wajar.** Mampu mengevaluasi sebuah beatmap dengan wajar, baik dengan mencoba memainkannya ataupun pengalaman modding, adalah bagian inti dari kemampuan menilai kualitas sebuah beatmap. Jika kamu dianggap tidak memiliki keterampilan bermain atau pengalaman modding yang memadai, kami mungkin mempertanyakan kemampuanmu dala ...
```
```suggestion
- **Jangan menominasi sebuah beatmap jika kamu tidak dapat mengevaluasinya dengan wajar.** Mampu mengevaluasi sebuah beatmap dengan wajar, baik dengan mencoba memainkannya ataupun pengalaman modding, adalah bagian inti dari kemampuan menilai kualitas sebuah beatmap. Jika kamu dianggap tidak memiliki keterampilan bermain atau pengalaman modding yang memadai, kami mungkin mempertanyakan kemampuanmu dalam menilai sebuah beatmap.
```

rorre commented on wiki/People/The_Team/Beatmap_Nominators/Rules/id.md:
```diff
@@ -0,0 +1,76 @@
+- **Jangan menominasi sebuah beatmap jika kamu tidak yakin terhadap sesuatu.** Dalam kasus seperti itu, mintalah bantuan dari sesama nominator sebelum membuat keputusan yang terburu-buru. Ini termasuk mode permainan yang tidak dikenal, sesuatu yang mencurigakan/masalah unrankable, masalah kualitas atau timing yang rumit.
+- **Jangan menominasi sebuah beatmap jika kamu tidak dapat meninainya dengan wajar.** Mampu mengevaluasi sebuah beatmap dengan wajar, baik dengan mencoba memainkannya ataupun pengalaman modding, adalah bagian inti dari kemampuan menilai kualitas sebuah beatmap. Jika kamu dianggap tidak memiliki keterampilan bermain atau pengalaman modding yang memadai, kami mungkin mempertanyakan kemampuanmu dala ...
+- **Jangan menominasi beatmap dari mode permainan yang tidak sesuai dengan spesialisasi kamu.** Kamu harus memenuhi syarat untuk menominasi setidaknya salah satu dari mode permainan yang termasuk dalam beatmapset dengan manjadi anggota dari masing-masing bagian pada Beatmap Nominator agar diizinkan untuk menominasinya.
+- **Pastikan tidak ada postingan mod dalam diskusi beatmap yang diabaikan sebelum menominasi sebuah beatmap.** Postingan tanpa balasan yang memadai dapat dianggap sebagai postingan yang diabaikan. Jika ragu, periksa kembali apakah Kode Etik Mapping dan Modding telah dilanggar di diskusi beatmap dan lakukanlah tindakan yang sesuai.
```
```suggestion
- **Pastikan tidak ada postingan mod dalam diskusi beatmap yang diabaikan sebelum menominasi sebuah beatmap.** Postingan tanpa balasan yang memadai dapat dianggap sebagai postingan yang diabaikan. Jika ragu, periksa kembali apakah Kode Etik Mapping dan Modding telah dilanggar di diskusi beatmap dan lakukan tindakan yang sesuai.
```

rorre commented on wiki/People/The_Team/Beatmap_Nominators/Rules/id.md:
```diff
@@ -0,0 +1,76 @@
+
+### Kualifikasi
+
+- **Beatmap yang berbeda dari versi yang sama dari sebuah lagu tidak dapat dikualifikasikan di mode yang sama pada waktu yang bersamaan.** Versi yang sama dari sebuah lagu seperti potongan, perpanjangan, dan cover sederhana.
```
```suggestion
- **Beatmap yang berbeda dari versi sebuah lagu yang sama tidak dapat dikualifikasikan di mode yang sama pada waktu yang bersamaan.** Versi yang sama dari sebuah lagu yang dimaksud adalah potongan, perpanjangan, dan cover sederhana.
```

rorre commented on wiki/People/The_Team/Beatmap_Nominators/Rules/id.md:
```diff
@@ -0,0 +1,76 @@
+### Kualifikasi
+
+- **Beatmap yang berbeda dari versi yang sama dari sebuah lagu tidak dapat dikualifikasikan di mode yang sama pada waktu yang bersamaan.** Versi yang sama dari sebuah lagu seperti potongan, perpanjangan, dan cover sederhana.
+- **Hanya 3 versi lagu yang berbeda yang dapat dikualifikasikan di mode yang sama pada waktu yang bersamaan.** Versi yang berbeda seperti remix, kompilasi, dan mashup.
```
```suggestion
- **Hanya 3 versi lagu yang berbeda yang dapat dikualifikasikan di mode yang sama pada waktu yang bersamaan.** Versi yang berbeda yang dimaksud adalah remix, kompilasi, dan mashup.
```

rorre commented on wiki/People/The_Team/Beatmap_Nominators/Rules/id.md:
```diff
@@ -0,0 +1,76 @@
+
+- **Hanya anggota full yang dapat mendiskualifikasi beatmap, apapun alasannya.** Namun, anggota probationary masih dapat berpartisipasi dalam diskusi tersebut dan melaporkan beatmap seperti yang dilakukan orang lain.
+- **Diskualifikasi sebaiknya dilakukan untuk beatmap yang mempunyai masalah unrankable, atau jika pemilik beatmapset tersebut memintanya.**
+- **Permintaan diskualifikasi oleh pemilik beatmapset dapat segera diterapkan.** Ini termasuk diri kamu sendiri, misalnya kamu ingin melakukan dq sendiri.
```
```suggestion
- **Permintaan diskualifikasi oleh pemilik beatmapset dapat segera diterapkan.** Ini termasuk diri kamu sendiri, misalnya kamu ingin melakukan diskualifikasi sendiri.
```
I am aware that EN ver uses `self-dq`, but uses `dq` here may cause confusion instead,

hikikuman replied:
ehh iya ya, awalnya sebenernya mau buat DQ huruf besar tapi jadinya huruf kecil entah mengapa. tapi bener juga, DQ kurang umum kalo buat orang awam.

rorre commented on wiki/People/The_Team/Beatmap_Nominators/Rules/id.md:
```diff
@@ -0,0 +1,76 @@
+Silahkan lihat halaman [beatmap veto](/wiki/People/The_Team/Beatmap_Nominators/Beatmap_Veto) untuk informasi lengkap tentang proses veto.
+
+- **Jangan memveto beatmap dari mode permainan yang tidak sesuai dengan spesialisasi kamu.** Kamu tidak diizinkan untuk menominasinya, jadi menghalangi nominasi tanpa kemampuan untuk menominasi kembali beatmap tersebut tidak masuk akal. Hal ini tidak berlaku untuk veto yang terkait dengan audio, background, atau konten lain yang tidak secara langsung berhubungan dengan mode permainan dari beatmap ...
+- **Kamu dapat memveto beatmapset yang sudah dinominasi/dikualifikasi dengan memposting masalah yang menurut kamu perlu diselesaikan sebelum mencapai status Ranked dalam bentuk masalah di diskusi modding terkait.** Ini akan mereset semua nominasi yang telah diterima oleh beatmapset dan menghentikannya dari proses menuju Ranked. Nominator lain tidak akan dapat menominasinya lagi sampai veto dicabu ...
```
```suggestion
- **Kamu dapat memveto beatmapset yang sudah dinominasi/dikualifikasi dengan memposting masalah yang menurut kamu perlu diselesaikan sebelum mencapai status Ranked dalam bentuk masalah di diskusi modding terkait.** Ini akan mereset semua nominasi yang telah diterima oleh beatmapset dan menghentikannya dari proses menuju Ranked. Nominator lain tidak akan dapat menominasinya lagi sampai veto dicabut. Postingan veto harus disertai masalah dan alasan yang jelas untuk menghentikan proses ranking beatmapset tersebut. Kamu harus berpartisipasi dalam diskusi dan berusaha mencapai kesepakatan bersama.
```

rorre commented on wiki/People/The_Team/Beatmap_Nominators/Rules/id.md:
```diff
@@ -0,0 +1,76 @@
+
+- **Jangan memveto beatmap dari mode permainan yang tidak sesuai dengan spesialisasi kamu.** Kamu tidak diizinkan untuk menominasinya, jadi menghalangi nominasi tanpa kemampuan untuk menominasi kembali beatmap tersebut tidak masuk akal. Hal ini tidak berlaku untuk veto yang terkait dengan audio, background, atau konten lain yang tidak secara langsung berhubungan dengan mode permainan dari beatmap ...
+- **Kamu dapat memveto beatmapset yang sudah dinominasi/dikualifikasi dengan memposting masalah yang menurut kamu perlu diselesaikan sebelum mencapai status Ranked dalam bentuk masalah di diskusi modding terkait.** Ini akan mereset semua nominasi yang telah diterima oleh beatmapset dan menghentikannya dari proses menuju Ranked. Nominator lain tidak akan dapat menominasinya lagi sampai veto dicabu ...
+- **Jangan memveto karena masalah unrakable.** Masalah unrankable berarti tidak boleh ada nominator yang menominasi dan mapper tersebut tidak punya pilihan lain selain memperbaikinya, sama halnya seperti sebuah penegakkan veto.
```
```suggestion
- **Jangan memveto karena masalah unrankable.** Masalah unrankable berarti tidak boleh ada nominator yang menominasi dan mapper tersebut tidak punya pilihan lain selain memperbaikinya, sama halnya seperti sebuah veto yang ditegakkan.
```

rorre commented on wiki/Modding/Timestamp/id.md:
```diff
@@ -0,0 +1,25 @@
+## Format
+
+Sintaks timestamp: `menit:detik:milidetik (angka kombo)`
+Sintaks link: `osu://edit/timestamp`
```
I'd say to keep using `Syntax` as `Sintaks` isn't an actual word in KBBI. And due to it's common nature I believe `Syntax` is easier to understand rather than `Sintaks`.

hikikuman replied:
should it be changed into `Timestamp syntax`? or, just change `Sintaks` to `Syntax` (it will be `Syntax timestamp`, same as en.md)

hikikuman replied:
aight, changed.

rorre commented on wiki/Modding/Star_priority/id.md:
```diff
@@ -0,0 +1,19 @@
+   - Sebuah bintang kudosu dapat diberikan oleh pengguna ke beatmap mana pun, sebuah bintang kudosu yang diberikan harus dibayar dengan satu kudosu.
+   - Rasio konversi sebelumnya adalah tiga kudosu untuk satu bintang kudosu yang meningkatkan Star Priority menjadi dua.
+   - Semakin tinggi star prioritynya, semakin tinggi posisi utas beatmap pada subforum Works in Progress atau Pending.
+   - Diperlukan setidaknya 12  star priority agar sebuah beatmap dapat di bubbled atau di ranked.
```
```suggestion
   - Diperlukan setidaknya 12 star priority agar sebuah beatmap dapat di bubbled atau di ranked.
```

rorre commented on wiki/Modding/Star_priority/id.md:
```diff
@@ -0,0 +1,19 @@
+   - Diperlukan setidaknya 12  star priority agar sebuah beatmap dapat di bubbled atau di ranked.
+2. Suatu sistem untuk menentukan apakah suatu Permintaan Fitur diinginkan oleh komunitas.
+   - Semakin tinggi star prioritynya, semakin cepat permintaan tersebut mendapat perhatian.
+   - Pengguna yang sedang memiliki osu! supporter saat pemungutan suara dapat memberikan sebuah bintang berwarna merah muda (![Kudosu](/wiki/shared/Kudos.png)) yang meningkatkan Star Priority masing-masing 2, sedangkan pengguna yang tidak memiliki osu! supporter dapat memberikan sebuah bintang berwarna biru (![Star](/wiki/shared/icon/star.gif)) yang masing-masing hanya memiliki nilai 1. Pengguna  ...
```
```suggestion
   - Pengguna yang sedang memiliki osu! supporter saat pemungutan suara dapat memberikan sebuah bintang berwarna merah muda (![Kudosu](/wiki/shared/Kudos.png)) yang meningkatkan Star Priority masing-masing 2, sedangkan pengguna yang tidak memiliki osu! supporter dapat memberikan sebuah bintang berwarna biru (![Star](/wiki/shared/icon/star.gif)) yang masing-masing hanya memiliki nilai 1. Pengguna dapat memberikan beberapa bintang tergantung pada seberapa banyak sisa vote mereka.
```

rorre commented on wiki/Modding/Proto-bubble/id.md:
```diff
@@ -0,0 +1,12 @@
+
+# Proto-bubble
+
+**Proto-bubble** (***pubble***) adalah sebuah ikon pada postingan forum yang digunakan oleh anggota [Mapping Assistance Team](/wiki/Modding/Mapping_Assistance_Team) (*MAT*) pada urutan sebelumnya sampai masuk ke [proses ranking beatmap](/wiki/Beatmap_ranking_procedure). Ikon ini dirancang untuk menunjukkan [beatmap](/wiki/Beatmaps) yang diyakini memiliki kualitas yang baik untuk diberi [ranking]( ...
```
```suggestion
**Proto-bubble** (***pubble***) adalah sebuah ikon pada postingan forum yang digunakan oleh anggota [Mapping Assistance Team](/wiki/Modding/Mapping_Assistance_Team) (*MAT*) pada urutan sebelumnya sampai masuk ke [proses ranking beatmap](/wiki/Beatmap_ranking_procedure). Ikon ini dirancang untuk menunjukkan [beatmap](/wiki/Beatmaps) yang diyakini memiliki kualitas yang baik untuk diberi [ranking](/wiki/Beatmaps#ranked) oleh anggota [Beatmap Appreciation Team](/wiki/Modding/Beatmap_Appreciation_Team) (*BAT*), yang kemudian dapat memberikan sebuah ikon full bubble<!-- TODO: bubble stub --> jika beatmap tersebut memenuhi persyaratan mereka.
```

rorre commented on wiki/Modding/NAT_meetings/id.md:
```diff
@@ -0,0 +1,40 @@
+
+# Rapat terbuka NAT
+
+**Rapat terbuka NAT** adalah diskusi terarah yang diselenggarakan oleh [Nomination Assessment Team](/wiki/People/The_Team/Nomination_Assessment_Team) (*NAT*) di channel `#modding` pada [server Discord osu!dev](/wiki/osu!dev_Discord_server). Topik rapat berkaitan dengan ekosistem [mapping](/wiki/Beatmapping) dan [modding](/wiki/Modding), termasuk masalah, solusi yang berpotensi, ide baru, dan lapo ...
```
```suggestion
**Rapat terbuka NAT** adalah diskusi terarah yang diselenggarakan oleh [Nomination Assessment Team](/wiki/People/The_Team/Nomination_Assessment_Team) (*NAT*) di channel `#modding` pada [server Discord osu!dev](/wiki/osu!dev_Discord_server). Topik rapat berkaitan dengan ekosistem [mapping](/wiki/Beatmapping) dan [modding](/wiki/Modding), termasuk masalah, potensi solusi, ide baru, dan laporan perkembangan. Meskipun kontributor terbanyak dalam diskusi ini adalah anggota NAT atau [Beatmap Nominator](/wiki/People/The_Team/Beatmap_Nominators), siapapun diperbolehkan untuk berpartisipasi.
```

rorre commented on wiki/Modding/Nuke/id.md:
```diff
@@ -0,0 +1,21 @@
+
+# Nuke
+
+**Nuke** adalah sebuah alat moderasi [beatmap](/wiki/Beatmaps) di tahun-tahun awal osu!. Beatmap apa pun yang dianggap mengabaikan aturan (atau kreator mengabaikan [Kode Etik](/wiki/Rules/Code_of_Conduct_for_Modding_and_Mapping)) bisa di nuke oleh [Beatmap Appreciation Team (BAT)](/wiki/Modding/Beatmap_Appreciation_Team) atau [Global Moderation Team (GMT)](/wiki/People/The_Team/Global_Moderation_ ...
```
```suggestion
**Nuke** adalah sebuah alat moderasi [beatmap](/wiki/Beatmaps) di tahun-tahun awal osu!. Beatmap apa pun yang dianggap mengabaikan aturan (atau kreator mengabaikan [Kode Etik](/wiki/Rules/Code_of_Conduct_for_Modding_and_Mapping)) bisa di-nuke oleh [Beatmap Appreciation Team (BAT)](/wiki/Modding/Beatmap_Appreciation_Team) atau [Global Moderation Team (GMT)](/wiki/People/The_Team/Global_Moderation_Team).
```
verb shouldnt be separated

hikikuman replied:
ahh ya lupa pake strip. didn't notice this :D nanti semuanya di recheck lagi

rorre commented on wiki/Modding/Nuke/id.md:
```diff
@@ -0,0 +1,21 @@
+
+Saat sebuah beatmap di nuke, akan muncul simbol radioaktif ![](/wiki/shared/icon/nuke.gif) pada utas beatmap tersebut. Simbol tersebut menjadi tanda bagi anggota BAT/GMT lainnya bahwa beatmap tersebut tidak dapat dipertimbangkan untuk [Ranked](/wiki/Beatmap_ranking_procedure#ranked) sampai perubahan signifikan telah dibuat. Jika perubahan yang sesuai sudah dilakukan, ikon tersebut akan dihapus da ...
+
+Beatmap dapat di nuke karena berbagai alasan, tetapi yang paling umum adalah:
```
```suggestion
Beatmap dapat di-nuke karena berbagai alasan, tetapi yang paling umum adalah:
```

rorre commented on wiki/Modding/id.md:
```diff
@@ -1,38 +1,40 @@
+Untuk melakukan mod pada sebuah beatmapset, mulailah dengan mengunduh versi terbaru dan buka [halaman diskusi](/wiki/Beatmap_Discussion)nya. Mainkan tingkat kesulitan yang ingin kamu mod dan ingatlah masalah apa saja yang kamu temukan. Setelah selesai, buka tingkat kesulitan tersebut di editor dan lakukan analisis untuk menemukan masalah lain.
 
-**Kelima**, silahkan post mod yang sudah kita buat.
+Saat kamu menemukan masalah, pilih objek hit tersebut, lalu gunakan pintasan salin (`Ctrl` + `C`) untuk menyalin informasi dari timestamp sebagai teks. Pada diskusi beatmap, tempel (`Ctrl` + `V`) timestamp sekaligus umpan balik dari kamu. Tab `Timeline` digunakan untuk masalah yang berkaitan dengan bagian tertentu dari sebuah beatmap, tab `Umum (Tingkat Kesulitan ini)` digunakan untuk umpan balik ...
```
```suggestion
Saat kamu menemukan masalah, pilih objek hit tersebut, lalu gunakan pintasan salin (`Ctrl` + `C`) untuk menyalin informasi dari timestamp sebagai teks. Pada diskusi beatmap, tempel (`Ctrl` + `V`) timestamp sekaligus umpan balik dari kamu. Tab `Timeline` digunakan untuk masalah yang berkaitan dengan bagian tertentu dari sebuah beatmap, tab `Umum (Tingkat kesulitan ini)` digunakan untuk umpan balik pada bagian yang lebih besar dari sebuah beatmap, dan tab `Umum (Semua tingkat kesulitan)` digunakan untuk masalah yang berlaku untuk semua tingkat kesulitan pada sebuah beatmapset.
```
follow osu!site capitalization

hikikuman replied:
aight, my bad

rorre commented on wiki/Modding/id.md:
```diff
@@ -1,38 +1,40 @@
+Saat kamu menemukan masalah, pilih objek hit tersebut, lalu gunakan pintasan salin (`Ctrl` + `C`) untuk menyalin informasi dari timestamp sebagai teks. Pada diskusi beatmap, tempel (`Ctrl` + `V`) timestamp sekaligus umpan balik dari kamu. Tab `Timeline` digunakan untuk masalah yang berkaitan dengan bagian tertentu dari sebuah beatmap, tab `Umum (Tingkat Kesulitan ini)` digunakan untuk umpan balik ...
 
-## Hal-Hal Yang Perlu Diperhatikan Dalam Modding
+Semua postingan mod harus membahas apa saja masalahnya, mengapa itu menjadi masalah, dan bagaimana cara memperbaikinya. Lihat Kode etik dalam Modding dan Mapping](/wiki/Rules/Code_of_Conduct_for_Modding_and_Mapping?locale=id#membuat-postingan-mod) untuk detail lebih lanjut tentang cara membuat format postingan mod.
```
```suggestion
Semua postingan mod harus membahas apa saja masalahnya, mengapa itu menjadi masalah, dan bagaimana cara memperbaikinya. Lihat [Kode etik dalam Modding dan Mapping](/wiki/Rules/Code_of_Conduct_for_Modding_and_Mapping?locale=id#membuat-postingan-mod) untuk detail lebih lanjut tentang cara membuat format postingan mod.
```

rorre commented on wiki/Modding/id.md:
```diff
@@ -1,38 +1,40 @@
-- Setelah folder check, hal yang paling sering dilakukan adalah mencek kesalahan pada info field lagu. Misalnya nama artis, judul lagu, source, dan tags.
-- Ingat bahwa semua info field pada lagu harus sama. Dan beritahu mapper jika mapper mengulang artis dan source pada tag.
-- Setelah itu hal yang paling mudah dilakukan adalah mengecek konsistensi timing, seperti BPM dan offset, pada tiap tingkat kesulitan.
+Beatmapper biasanya memberikan [kudosu](/wiki/Glossary#kudosu!) ke postingan mod yang mereka anggap berguna dengan mengeklik tombol *jempol*. Statistik ini dapat dilacak pada profil setiap pengguna dan menjadi rujukan saat melamar untuk bergabung dengan [Beatmap Nominator](/wiki/People/The_Team/Beatmap_Nominators).
```
```suggestion
Beatmapper biasanya memberikan [kudosu](/wiki/Glossary#kudosu!) ke postingan mod yang mereka anggap berguna dengan mengeklik tombol ibu jari. Statistik ini dapat dilacak pada profil setiap pengguna dan menjadi rujukan saat melamar untuk bergabung dengan [Beatmap Nominator](/wiki/People/The_Team/Beatmap_Nominators).
```
dunno why is there the need to use jempol tbh xD

hikikuman replied:
wkwkwkwk sebenernya sama aja sih dua2nya

rorre reviewed (changes requested):
Overall looks alright...?

There are some inconsistencies with `Anda` and `kamu`, when it's formal (ex: Rules), use `Anda`, but when it's not, use `kamu`.

hikikuman commented on wiki/People/The_Team/Beatmap_Nominators/Beatmap_Veto/id.md:
```diff
@@ -0,0 +1,48 @@
+# Beatmap veto
+
+_Beatmap veto_ memungkinkan seorang [Beatmap Nominator](/wiki/People/Beatmap_Nominators) untuk menahan sebuah beatmap dari Kualifikasi jika mereka merasa terdapat masalah yang signifikan terkait kualitas beatmap yang membuatnya tidak cocok untuk bagian Ranked. Mencari diskusi atau klarifikasi lebih lanjut tentang segala jenis masalah kualitas yang mereka rasa perlu ditangani sebelum map tersebut  ...
```
okokokok

hikikuman commented on wiki/People/The_Team/Beatmap_Nominators/Beatmap_Veto/id.md:
```diff
@@ -0,0 +1,48 @@
+
+_Beatmap veto_ memungkinkan seorang [Beatmap Nominator](/wiki/People/Beatmap_Nominators) untuk menahan sebuah beatmap dari Kualifikasi jika mereka merasa terdapat masalah yang signifikan terkait kualitas beatmap yang membuatnya tidak cocok untuk bagian Ranked. Mencari diskusi atau klarifikasi lebih lanjut tentang segala jenis masalah kualitas yang mereka rasa perlu ditangani sebelum map tersebut  ...
+
+Semua Beatmap Nominator dapat mengajukan sebuah beatmap veto, asalkan mereka secara resmi memenuhi syarat untuk mode permainan yang dimaksud, sebagaimana dinyatakan dalam [Peraturan Beatmap Nominator](/wiki/People/Beatmap_Nominators/Rules). Setiap beatmap veto harus didiskusikan oleh kedua belah pihak - mapper serta Nominator mengajukan beatmap veto - dalam upaya mencapai kesepakatan yang disepak ...
```
yea, ok. my bad D: kayanya itu kurang yang sih, tapi pake pengaju lebih simpel

hikikuman commented on wiki/People/The_Team/Beatmap_Nominators/Becoming_a_Beatmap_Nominator/id.md:
```diff
@@ -0,0 +1,56 @@
+
+Berikut ini adalah modding ciri-ciri yang diharapkan dari Beatmap Nominator. Kegagalan untuk menampilkan salah satu dari ciri-ciri ini dapat mengakibatkan lamaran ditolak.
+
+- Ulasan pada semua tingkat kesulitan
```
ahh i see, okok

hikikuman commented on wiki/People/The_Team/Beatmap_Nominators/Becoming_a_Beatmap_Nominator/id.md:
```diff
@@ -0,0 +1,56 @@
+
+- Ini tergantung pada total `Skor Mod` yang dicapai pelamar dalam beberapa bulan terakhir. Nilai ini ditentukan menurut rumus berikut:
+
+![Mod Score Formula](/wiki/shared/Modscore_new_wiki.png "Mod Score Formula")
```
ahh ya lupa ini D: maaf

hikikuman commented on wiki/People/The_Team/Beatmap_Nominators/General_Information/id.md:
```diff
@@ -0,0 +1,49 @@
+- [Kode Etik untuk Mapping dan Modding](/wiki/Ranking_Criteria/Code_of_Conduct): Sebagai seorang Beatmap Nominator, kamu diharapkan dapat menjadi sosok yang patut dicontoh di komunitas, seseorang yang orang lain hormati dan hormati, jadi pelanggaran dapat membuat kamu ditegur atau bahkan dikeluarkan.
+- [Ranking Criteria](/wiki/Ranking_Criteria): Jika kamu sudah menjadi seorang Beatmap Nominator, kamu harus terbiasa dengan ini. Pada dasarnya, ini berfungsi untuk menjelaskan aspek mapping apa yang diperbolehkan dan tidak diperbolehkan di bagian Ranked, serta memasukkan beberapa pedoman umum yang orang harus coba ikuti dalam kebanyakan kasus.
+- [Informasi Beatmap Veto](/wiki/People/Beatmap_Nominators/Beatmap_Veto): Untuk beatmap dengan masalah kualitas secara objektif yang sebetulnya tidak melanggar aturan dan pedoman dari Ranking Criteria, tetapi yang menurut Anda membuat beatmap tersebut tidak sesuai untuk bagian Ranked, beatmap veto memungkinkan Anda menahan beatmap dari Kualifikasi.
+- [Mapset Verifier](https://osu.ppy.sh/community/forums/topics/943895): Sebuah alat yang dibuat oleh anggota NAT yaitu [Naxess](https://osu.ppy.sh/users/8129817). Alat ini memeriksa berbagai masalah unrankable, serta memiliki fitur snapshot agar lebih mudah dalam membandingkan versi dari map yang bermanfaat saat pengecekan ulang. **Ini seharusnya tidak menggantikan pemeriksaan beatmap sendiri**,  ...
```
hmm you're right. okkokook

SeverinoAji commented on wiki/People/The_Team/Beatmap_Nominators/Beatmap_Veto/id.md:
```diff
@@ -0,0 +1,48 @@
+# Beatmap veto
+
+*Beatmap veto* memungkinkan seorang [Beatmap Nominator](/wiki/People/Beatmap_Nominators) untuk menahan sebuah beatmap dari Kualifikasi jika mereka merasa terdapat masalah yang signifikan terkait kualitas beatmap yang membuatnya tidak cocok untuk bagian Ranked. Mencari diskusi atau klarifikasi lebih lanjut tentang segala jenis masalah kualitas yang mereka rasa perlu ditangani sebelum map tersebut  ...
```
```suggestion
*Beatmap veto* memungkinkan seorang [Beatmap Nominator](/wiki/People/Beatmap_Nominators) untuk menahan sebuah beatmap dari sesi Kualifikasi jika mereka mendapati masalah signifikan terkait kualitas beatmap yang membuatnya belum sesuai untuk menjadi bagian dari beatmap dengan status Ranked. Mendiskusikan atau mengklarifikasi lebih lanjut tentang segala jenis masalah kualitas yang mereka rasa perlu ditangani sebelum map tersebut dapat dilanjutkan ke sesi Kualifikasi adalah salah satu tanggung jawab utama mereka.
```

SeverinoAji commented on wiki/People/The_Team/Beatmap_Nominators/Beatmap_Veto/id.md:
```diff
@@ -0,0 +1,48 @@
+
+*Beatmap veto* memungkinkan seorang [Beatmap Nominator](/wiki/People/Beatmap_Nominators) untuk menahan sebuah beatmap dari Kualifikasi jika mereka merasa terdapat masalah yang signifikan terkait kualitas beatmap yang membuatnya tidak cocok untuk bagian Ranked. Mencari diskusi atau klarifikasi lebih lanjut tentang segala jenis masalah kualitas yang mereka rasa perlu ditangani sebelum map tersebut  ...
+
+Semua Beatmap Nominator dapat mengajukan sebuah beatmap veto, asalkan mereka secara resmi memenuhi syarat untuk mode permainan yang dimaksud, sebagaimana dinyatakan dalam [Peraturan Beatmap Nominator](/wiki/People/Beatmap_Nominators/Rules). Setiap beatmap veto harus didiskusikan oleh kedua belah pihak - mapper serta Nominator pengaju beatmap veto - dalam upaya mencapai kesepakatan yang disepakati ...
```
```suggestion
Semua Beatmap Nominator dapat mengajukan sebuah beatmap veto, asalkan mereka secara resmi memenuhi syarat untuk mode permainan yang dimaksud, sebagaimana dinyatakan dalam [Peraturan Beatmap Nominator](/wiki/People/Beatmap_Nominators/Rules). Setiap beatmap veto harus didiskusikan oleh kedua belah pihak - mapper serta Nominator yang mengajukan beatmap veto - dalam upaya mencapai kesepakatan.
```

hikikuman commented on wiki/People/The_Team/Beatmap_Nominators/Beatmap_Veto/id.md:
```diff
@@ -0,0 +1,48 @@
+
+*Beatmap veto* memungkinkan seorang [Beatmap Nominator](/wiki/People/Beatmap_Nominators) untuk menahan sebuah beatmap dari Kualifikasi jika mereka merasa terdapat masalah yang signifikan terkait kualitas beatmap yang membuatnya tidak cocok untuk bagian Ranked. Mencari diskusi atau klarifikasi lebih lanjut tentang segala jenis masalah kualitas yang mereka rasa perlu ditangani sebelum map tersebut  ...
+
+Semua Beatmap Nominator dapat mengajukan sebuah beatmap veto, asalkan mereka secara resmi memenuhi syarat untuk mode permainan yang dimaksud, sebagaimana dinyatakan dalam [Peraturan Beatmap Nominator](/wiki/People/Beatmap_Nominators/Rules). Setiap beatmap veto harus didiskusikan oleh kedua belah pihak - mapper serta Nominator pengaju beatmap veto - dalam upaya mencapai kesepakatan yang disepakati ...
```
sblmnya itu kurang yang, tapi diganti jadi `pengaju`. barusan cek di kbbi sih nggak ada `pengaju`, jadi ganti ke `yang mengajukan` aja

SeverinoAji commented on wiki/People/The_Team/Beatmap_Nominators/Beatmap_Veto/id.md:
```diff
@@ -0,0 +1,48 @@
+
+## Melakukan sebuah beatmap veto
+
+Sebagai seorang Beatmap Nominator, kamu bisa memveto sebuah beatmap melalui salah satu dari dua cara berikut:
```
```suggestion
Sebagai seorang Beatmap Nominator, kamu bisa melakukan veto kepada sebuah beatmap melalui salah satu dari dua cara berikut:
```

hikikuman replied:
diliat dari KBBI, `memveto` should be fine

SeverinoAji commented on wiki/People/The_Team/Beatmap_Nominators/Beatmap_Veto/id.md:
```diff
@@ -0,0 +1,48 @@
+Sebagai seorang Beatmap Nominator, kamu bisa memveto sebuah beatmap melalui salah satu dari dua cara berikut:
+
+- Dalam kasus beatmapset yang dinominasi, reset nominasinya.
+- Dalam kasus beatmapset yang dikualifikasi, diskualifikasikan.
```
```suggestion
- Dalam kasus beatmapset yang berada pada sesi kualifikasi, diskualifikasikan.
```

hikikuman replied:
oke applied

hikikuman commented:
Re-added all commits.

hikikuman commented:
I've adjusted the translation based on https://github.com/ppy/osu-wiki/pull/4050/. So, I just added Modding/* (BN Tiers, Delegation of BADASS, QAT, QAT Leaders), and Beatmap_Nominators/* (BN Test) to the PR. Also, update Modding/* (BAT, and BATm), and Beatmap_Nominators/* (Becoming a Beatmap Nominator).

SeverinoAji commented on wiki/Modding/Delegation_of_the_Beatmap_Approval_Deciding_Authorities_Submission_Squad/id.md:
```diff
@@ -0,0 +1,9 @@
+
+**Delegasi Beatmap Approval Deciding Authorities; Submission Squad (*BADASS*)** adalah metode seleksi berbasis forum pertama untuk anggota baru [Beatmap Appreciation Team (*BAT*)](/wiki/Modding/Beatmap_Appreciation_Team). Sebelumnya, anggota baru ditambahkan melalui pemungutan suara informal di channel [IRC](/wiki/Internet_Relay_Chat) permainan `#bat`, yang tidak menerima visibilitas yang cukup.
+
+Inisiatif ini dipimpin oleh [Ivalset](https://osu.ppy.sh/users/827), seorang [BAT manager](/wiki/Modding/BAT_Managers) pada saat itu, dan seudah berjalan selama sebilan siklus dari 9 Maret 2009 sampai dengan 14 Oktober 2009.
```
```suggestion
Inisiatif ini dipimpin oleh [Ivalset](https://osu.ppy.sh/users/827), seorang [BAT manager](/wiki/Modding/BAT_Managers) pada saat itu, dan seudah berjalan selama sembilan siklus dari 9 Maret 2009 sampai dengan 14 Oktober 2009.
```

hikikuman commented:
Changed `Ranking Criteria`, `ranking criteria`, and `RC` to `Kriteria Ranking` or `kriteria ranking`, based on https://github.com/ppy/osu-wiki/pull/4591
